forest/sycamore/base.py

```diff
@@ -91,6 +91,13 @@ def compute_survey_stats(
     Returns:
         True if successful, False otherwise
     """
+
+    if submits_timeframe not in [
+        Frequency.HOURLY_AND_DAILY, Frequency.HOURLY, Frequency.DAILY
+    ]:
+        logger.error("Error: Invalid submits timeframe")
+        return False
```

Comment:
Does this function need to return a `bool`? I don't see the return value being checked anywhere.

Reply:
I just used this statement: 

`Returns:
        True if successful, False otherwise
`

Reply:
Right. I was curious about the return value in general because this (https://github.com/onnela-lab/forest/blob/oak-minutely/forest/sycamore/__main__.py#L46) is the only place where this function is called and the return value is ignored. We can fix this later.